lua/net.lua

```diff
@@ -0,0 +1,26 @@
+local ffi = require("ffi")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- TODO Edit the network configuration data.
+local net_config = {
+	num_rx_queues = 1,
+	num_tx_queues = 1,
+}
+
+-- Function that sets up the network.
+function M.setup_block()
+	local ret = 0
+
+	-- Init the network configuration structure.
+	conf = gatekeeperc.alloc_net_conf()
+	conf.num_rx_queues = net_config.num_rx_queues
+	conf.num_tx_queues = net_config.num_tx_queues
+
+	-- Set up the network.
+	ret = gatekeeperc.gatekeeper_init_network(conf)
```

Comment:
You don't need variable `ret`, just write `return atekeeperc.gatekeeper_init_network(conf)`.